ArchiSteamFarm/Localization/Strings.resx

```diff
@@ -26,36 +26,36 @@
         <value>[base64 mime encoded string representing a byte array form of the .NET Framework object]</value>
         <comment>This is a comment</comment>
     </data>
-
-    There are any number of "resheader" rows that contain simple
+          
```

Comment:
You still have a lot of whitespace changes in this file. My advice, reset the file back to original and add only 4 lines you need.